sphinx_rtd_theme/layout.html

```diff
@@ -170,6 +170,12 @@
 
         {%- block navigation %}
         <div class="wy-menu wy-menu-vertical" data-spy="affix" role="navigation" aria-label="{{ _('Main') }}">
+          
+          {#- MOBILE NAV, TRIGGLES SIDE NAV ON TOGGLE #}
+          <div class="wy-close-top">
+            <i data-toggle="wy-nav-top" class="fa fa-close"></i>
```

Comment:
Is there a way to make this more accessible? I wasn't really sure what hitting the X would do when I tried to use it -- perhaps even some text along with the `X` like `Close Sidebar X`? I'm guessing there's also an `aria-label` we should be using here for screen readers?

Reply:
It looks like a close button is the example in these docs, so I think we should definitely include one: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-label_attribute#examples

Reply:
`aria-label` is the best solution here, yeah. Verbose text would fill out too much space and would need to be translated, which introduced other horizontal spacing concerns.

Reply:
I do wonder about something a bit more explicit that it's closing the menu, but I agree that translation point is a good one. Do the aria labels also need to be translated?

Reply:
Yeah, aria elements should also be translated. The theme actually has pretty active translations, so localization is definitely a bit more effective here.